### PR TITLE
FHG-3232 Request details page

### PR DESCRIPTION
**Pull request by MartinBelton-gov**
Also includes:
* remove the existing solo navigation menu item

### CODE REVIEW

**Review by lazcool — APPROVED** [review]
Approving @MartinBelton-gov 's contributions.

File: src/FamilyHubs.RequestForSupport.Core/ApiClients/ReferralClientService.cs [hunk 1]
```diff
@@ -47,19 +60,87 @@ public async Task<PaginatedList<ReferralDto>> GetRequestsForConnectionByOrganisa
...
+        var stringResult = await response.Content.ReadAsStringAsync();
+        return stringResult;
+    }
+
+    public async Task<string> UpdateReferralStatus(long referralId, ReferralStatus referralStatus, string? reason = null)
+    {
+        var request = new HttpRequestMessage
+        {
+            // copilot's suggestion. seems good
+            //Method = HttpMethod.Put,
```
Comment (MartinBelton-gov) [inline src/FamilyHubs.RequestForSupport.Core/ApiClients/ReferralClientService.cs:133]: Should this commented out code be removed

**Review by MartinBelton-gov — COMMENTED** [review]
@lazcool  Changes Approved

File: src/FamilyHubs.RequestForSupport.Core/ApiClients/ReferralClientService.cs (hunk 1 above)
Comment (lazcool) [reply src/FamilyHubs.RequestForSupport.Core/ApiClients/ReferralClientService.cs:133]: Changed it to a todo. Not a biggie though, so we can leave it for now.

File: src/FamilyHubs.RequestForSupport.Web/Pages/Index.cshtml.cs [hunk 2]
```diff
@@ -11,12 +11,16 @@ public class IndexModel : PageModel
...
         var user = HttpContext.GetFamilyHubsUser();
-        //todo: vcs admin or pro
-        if(user.Role == "VcsAdmin")
+        if(user.Role is RoleTypes.VcsProfessional or RoleTypes.VcsDualRole)
         {
-            return RedirectToPage("/VcsRequestForSupport/Dashboard");
+            return RedirectToPage("/Vcs/Dashboard");
         }
 
+        //if (user.Role is RoleTypes.LaProfessional or RoleTypes.LaDualRole)
```
[the remark this answers is not in the document]
Comment (lazcool) [reply src/FamilyHubs.RequestForSupport.Web/Pages/Index.cshtml.cs:19]: It's not needed yet, but it'll be needed when we add the progessional's dashboard.

**Pull request merged**